recipe/meta.yaml

```diff
@@ -139,6 +139,7 @@ requirements:
     - ncurses  # [not win]
     - libffi  # [not win]
     - libnsl  # [linux]
+    - libxcrypt  # [linux]
```

Comment:
@beckermr , just to confirm, is this what we want here? Is there anything else we need?

Reply:
Assuming it has a proper run export, this should be everything.

Reply:
@mbargull already converted the current python versions so you can compare to that recipe to be sure.

Reply:
That looks like what Marcel did in PR: https://github.com/conda-forge/python-feedstock/pull/664

Though would be happy to hear feedback from him

Looks like we already cover the `import crypt` test case

https://github.com/conda-forge/python-feedstock/blob/d9b57b0d84bc6772b2ec70dd73dbdc1536c6dc1e/recipe/run_test.py#L80

Reply:
I'm not against rebuilding against `libxcrypt`.
However, rebuilding this against an updated global pinning is probably not what would @AaronOpfer and others help.
We do not rebuild packages that depend on Python 3.6/3.7 anymore, which means that if we only rebuild `conda-forge::python=3.6,3.7`, we'll likely add updated dependency constraints which would be incompatible with Python extension modules on `conda-forge` that have been built against the same packages but only in older versions.
(Looking at the changes to `.ci_support/*.yaml` here, it doesn't look like much has changed, apart from the dropped `openssl=1.1.1` builds; but there could still be things like tightened `run_exports` in some of the dependencies -- i.e., someone should look at the actual constraints generated.)

Meaning, we might not want a full rerendering here to try to build against older libs (and have someone verify this by comparing with previous `python=3.6,3.7` builds).

So, to me `libcrypt`/`libxcrypt` is actually not the most important thing we should look out for here, but rather the other dependencies.

(FYI, Aaron's issue does probably not concern the actual linked `libcrypt` at all, but rather the fact that `Python.h` includes `crypt.h` for Python <=3.8; in >=3.9 it is only used for the `crypt` module and in >=3.13 it will not be used at all.)

Reply:
That's a good point Marcel!

We can revert the bulk of the variant changes (though some things like the Visual Studio version are out of our hands)

Also if you haven't already, please read over this discussion for more context: https://github.com/conda-forge/linux-sysroot-feedstock/issues/52#issuecomment-1946615469

Reply:
One more note: if CF maintainers would like to take over and directly edit this PR instead of puppeteering this conda-feedstock novice, please feel free to do so 🙂

Reply:
> I have consulted with my users and discovered they are still using the openssl 1.1.1 variant of Python 3.7.12

Um I hope not. OpenSSL 1.1.1 has already reached [EOL]( https://www.openssl.org/blog/blog/2023/09/11/eol-111/ )

> Do we believe we would fix this by modifying the recipe/conda_build_config.yaml and rerendering as the .ci_support/README suggests? Or would we hand-edit the files as we're doing artifact archeology on an old recipe?

Was thinking we could just

```
git checkout 3.7 -- .ci_support
git add .ci_support
git commit -m 'Restore `.ci_support` from `3.7`'
```

Reply:
> Um I hope not. OpenSSL 1.1.1 has already reached [EOL](https://www.openssl.org/blog/blog/2023/09/11/eol-111/)

Python 3.7 never officially gained support for OpenSSL 3. We only [did](https://github.com/conda-forge/python-feedstock/issues/516#issuecomment-942220263) that in conda-forge.